doc/classes/Image.xml

```diff
@@ -376,12 +376,12 @@
 			<argument index="2" name="color" type="Color">
 			</argument>
 			<description>
-				Sets the [Color] of the pixel at [code](x, y)[/code] if the image is unlocked. Example:
+				Sets the [Color] of the pixel at [code](x, y)[/code] if the image is locked. Example:
 				[codeblock]
 				var img = Image.new()
-				img.lock()
-				img.set_pixel(x, y, color) # Does not have an effect
 				img.unlock()
+				img.set_pixel(x, y, color) # Does not have an effect
+				img.lock()
```

Comment:
I think the typical workflow is lock + modify + unlock, so this example is a bit misleading. This would be better:
```
var img = Image.new()
img.lock()
img.set_pixel(x, y, color) # Works
img.unlock()
img.set_pixel(x, y, color) # Has no effect
```

BTW the description for `lock` should also be changed.